admin/app/services/admin/match_service.rb

```diff
@@ -3,7 +3,13 @@ class MatchService
     include Godmin::Resources::ResourceService
 
     attrs_for_index :id, :established, :newcomer, :status
-    attrs_for_show :established, :newcomer, :started_at, :concluded_at, :follow_up_mail_sent_at, :comment
+    attrs_for_show :comment,
```

Comment:
I think this affects the order the elements are rendered, maybe they were grouped like that to make sense in the interface?